package-lock.json

```diff
@@ -13505,7 +13505,7 @@
     "sinon": {
       "version": "9.0.2",
       "resolved": "https://registry.npmjs.org/sinon/-/sinon-9.0.2.tgz",
-      "integrity": "sha1-uQF+JGM/SxyY37bnhKXwUJ9f2F0=",
+      "integrity": "sha512-0uF8Q/QHkizNUmbK3LRFqx5cpTttEVXudywY9Uwzy8bTfZUhljZ7ARzSxnRHWYWtVTeh4Cw+tTb3iU21FQVO9A==",
```

Comment:
wait why was this file changed?

Reply:
not for sure if it does have to be reverted but if it does use this command:

 git checkout origin/dev package-lock.json

Reply:
package-lock changes are good. its means that a package was updated to patch bugs.